### ISSUE TITLE
M1 Simulator support

### ISSUE THREAD

**Issue by kei-sidorov** [dimelo/Dimelo-iOS#45]
Hello!

The library doesn't support an ARM simulator. When I build my project I got
```
ld: building for iOS Simulator, but linking in dylib built for iOS, file '<dir>/Pods/Dimelo-iOS/Dimelo.framework/Dimelo' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

You should use XCFramework instead of fat binary distribution, as Apple doesn't support it anymore.

**Comment by CedricBm** [comment]
Hello, sorry for the slow reply and thanks for the report. We'll handle it. For the record, our internal id is: RD-18883

**Comment by waelba** [comment]
Hi,

It's fixed in `v2.4.1`

Thanks
